src/app/views/contact_detail_view.js

```diff
@@ -0,0 +1,54 @@
+import Backbone from 'backbone';
+import _ from 'underscore';
+import $ from 'jquery';
+
+import Contact from '../models/contact.js';
+import Rolodex from '../collections/rolodex.js';
+import EditContactView from '../views/edit_contact_view.js';
+
+const ContactDetailView = Backbone.View.extend({
+  initialize: function(params) {
+    this.template = params.template;
+    this.listenTo(this.model, "change", this.render);
+  },
+
+  render: function() {
+    var compiledTemplate = this.template(this.model.toJSON());
+    this.$el.html(compiledTemplate);
+    return this;
+  },
+
+  events: {
+    "click button.btn-delete" : "removeCard",
+    "click button.btn-edit" : "editCard",
+    "click" : "preventClose"
+  },
+
+  preventClose: function(event) {
+    event.stopPropagation();
+  },
+
+  removeCard: function(event) {
+    event.stopPropagation();
+    this.model.destroy();
+    $("#contact-details").hide();
```

Comment:
Raw jQuery should be avoided in a view.  Instead this should be `this.$("#contact-details").hide();`